### PR TITLE
Change AREA ids and add function to remove areas

### LINKED ISSUE #108: Change AREA names in BD
## Related feature
Currently, the AREA names in DB area AREA1, AREA2, ... It creates lot of problems to identify them in the app.

## Improvement
Instead of the current way, we can create a uuid that the app will associate to the AREA before sending it to the server.
It will look like :
`
ca33c26a-a851-11ed-afa1-0242ac120002: {
    Action: {...},
    Reaction: {...}
}
`

The app would create the uuid before sending it to the server. You should modify a bit the interface concerning the Areas individually to be able to save the area id in the front. You just add a value to the object and you access/modify it when you have to use it :)

It will allow you to also delete an area server-side, as we will have an id that will be in the back-end and in the front-end.
To do this you should just create an api route that deletes an area using the user uid and the area uuid. And front-side, you should juste call this route with the required paramaters.

## Extra link (Optional)
_Add any other extra links_

## Screenshots (Optional)
_Add any screenshot_

### PR DESCRIPTION
**Pull request by JohanCDev**
# New feature
The AREAs are now distincts from their id (a uuid generated from the app and sent to the server). It allows us to handle more easily the AREAs while communicating between front and back.

This PR also adds the functionality of removing AREAs, linked to back-end.

# Updated feature

# Bug fixed

Closes #108

### CODE REVIEW

**Review by STom6 — DISMISSED** [review]
lgtm

File: Server/firebaseFunctions.js [hunk 1]
```diff
@@ -34,6 +34,15 @@ module.exports = {
...
     },
+    removeDataFromFireBase: function(path) {
+        return new Promise((resolve, reject) => {
+            database.ref(path).on('value', (snapshot) => {
+                if (snapshot.val()) {
+                    resolve(snapshot.ref.remove());
+                }
+            });
+        });
+    },
```
Comment (osvegn) [inline Server/firebaseFunctions.js]: missing documentation

**Review by osvegn — CHANGES_REQUESTED** [review]
Missing documentation

**Review by CedricCORGE — DISMISSED** [review]
lgtm

**Review by osvegn — DISMISSED** [review]

**Review by osvegn — APPROVED** [review]
Look good

**Review by 31Nathan — APPROVED** [review]
LGTM

**Pull request merged**